### ISSUE TITLE
Don't display if user is blocked by a user

### ISSUE THREAD

**Issue by Mg138** [sk22/megalodon#526]
The corresponding API endpoints have been (supposedly) removed as per [this issue](https://akkoma.dev/AkkomaGang/akkoma/issues/552).

This is possibly very low priority as it shouldn't affect usage, however should be a good cleanup task for any contributors c:

**Comment by sk22** [comment]
oh uh i didn't even know megalodon supports this ^^' (i suppose the functionality comes from mastodon for android?)

**Comment by sk22** [comment]
okay, apparently, there's no such api endpoint anyway - it's just a field called "blocked_by" that will always have "false" as a value, so i don't think i need to change anything on this end

**Comment by Mg138** [comment]
Ah, that's convenient.

Well the functionality that we've noticed was the fact that the follow button is greyed out if blocked_by is true, we assumed it was extra support for akkoma because we didn't notice such functionality on mastodon.

So is the reasoning to remove Akkoma's support for it, we'd also like to think such functionality isn't really worthy for any clients? Again, no rush to remove it or anything, just worth considering removing it.

**Comment by Mg138** [comment]
Mastodon might support it in the future, iunno, but like, it's very unnecessary information to show to the end user that you're blocked :/

**Comment by sk22** [comment]
i just tried it and mastodon (or glitch, at least) does seem to support it (and turns out mastodon for android supports it too). the web ui (of glitch) doesn't appear to be graying out the button on though (clicking the follow button still shows a "403 - action not allowed" pop-up - which makes sense).
and yep, the reasoning makes sense, i'll probably remove it then!
